submissions/medinatapanpa/html-css/dufuna2-1.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <title>Registration Page</title>
+</head>
+<body>
+ <img src="pistachios.jpg" alt="medicine" width="60" height="40">
+ <article width="40" >
+  Herbals is a Nigerian based herbal online store founded in 1984. Our herbal house sells various kinds of 
+  herbal spieces and medicine. Herbals offers a diverse option of herbs, from Nigerian, Mediterrenian and American 
+ to our customers. Customers experience an open option to diverse blends of herbs from all over the world in 
+ thier finest flavours.</article>
+<style>
+  form { align-self: auto;
+    }
+</style>
+ </P>
+    <form style="align-content: space-between;">
+      <fieldset>
+        <legend>REGISTER:</legend>
+                  <p>Please fill in this form to create an account.</p>
+                            
+                                     
+          <label for="firstname"><b>First Name</b></label>
+          <input type="text" placeholder="Enter First Name" name="firstname" required><br><p></p></th>
+                  
+                                
+          <label for="lastname"><b>Last Name</b></label>
+          <input type="text" placeholder="Last Name" name="lastname" required><br><p></p>
+
+          <label for="email"><b>Email</b></label>
+          <input type="text" placeholder="Enter Email" name="email" required><br><p></p>
+
+          <label for="psw"><b>Password</b></label>
+          <input type="password" placeholder="Enter Password" name="psw" required><br><p></p>
+      
+          <label for="psw-repeat"><b>Repeat Password</b></label>
+          <input type="password" placeholder="Repeat Password" name="psw-repeat" required><br><p></p>
+          
+          <button type="Cancel" class="cancelbtn">Cancel</button>
```

Comment:
Please, what's the cancel button for?

Reply:
Thank you. In case the person filling the form changes his/her mind ....do not want to proceed with the registration.

Reply:
ok

Reply:
Also, the naming of your file `dufuna2-1.html` just `dufuna.html` would have been better

Reply:
OK, thank you, i will effect the correction